docs/MigrationGuide.md

```diff
@@ -35,6 +38,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   It means that `LiteSource` also takes this parameter and have a new method `createTransformation`.
 * [#2635](https://github.com/TouK/nussknacker/pull/2635) `ContextInitializer.initContext` now takes `ContextIdGenerator` instead of `nodeId` and returns just a function
   with strategy of context initialization instead of serializable function with `Lifecycle`. To use it with Flink engine, use `FlinkContextInitializingFunction` wrapper.
+* [#2645](https://github.com/TouK/nussknacker/pull/2645) Simplify structure of available models (implementations of `ProcessConfigCreator`). `defaultModel.jar` and components 
+  should be used instead of custom implementations of `ProcessConfigCreator`, the only exception is when one wants to customize `ExpressionConfig`. 
```

Comment:
there should be info about renaming nu-flink-engine -> nu-flink-executor module

Reply:
:+1: done